libafl_v8/src/values.rs

```diff
@@ -0,0 +1,22 @@
+//! Value types for inputs passed to JavaScript targets
+
+use libafl::{
+    inputs::{BytesInput, HasBytesVec},
```

Comment:
where is `HasBytesVec` used 🤔

Reply:
It's a trait usage; the type is never referred to, but a type which implements its methods is used (and I used that method). Thus, the trait must be `use`d.

Reply:
ah, thx